Sources/Fullscreen/FullscreenGallery/FullscreenGalleryViewController.swift

```diff
@@ -179,24 +181,37 @@ public class FullscreenGalleryViewController: UIPageViewController {
     private func currentImageViewController() -> FullscreenImageViewController? {
         return viewControllers?.first as? FullscreenImageViewController
     }
+
+    // Find remainder of both positiv and negative numbers
+    private func remainder(_ number: Int, divider: Int) -> Int {
```

Comment:
Is it really the remainder you are calculating here? This is a quite confusing thing when dealing with negative number, quick googling didn't help me much either. Except that Swift `%` "modulo operand" actually is a remainder operation, but yeah, then it is the negative thing too 🤕 

Could we maybe just rename it to what you actually use it for instead? I guess something like `imageIndexForPage`?

Reply:
I might not have to think about the remainder of negative number after all, I'll check it out.

Reply:
What do you think about it now @osanoj ?

Reply:
Much better, now I don't have to scratch my head and think 😉